src/Daemon/BaseDaemon.cpp

```diff
@@ -846,6 +846,13 @@ void BaseDaemon::initialize(Application & self)
         DateLUT::setDefaultTimezone(config_timezone);
     }
 
+    if (config().has("prefer_system_tzdata"))
+    {
+        const bool prefer_system_tzdata = config().getBool("prefer_system_tzdata");
```

Comment:
would be nice to warn about that. But it's not the best place in the code to do some logging... 

Maybe we can read those envs & do some debug prints about TZ & prefer_system_tzdata somewhere in Server.cpp?